internal/backup_fetch_handler.go

```diff
@@ -65,33 +66,33 @@ func readRestoreSpec(path string, spec *TablespaceSpec) (err error) {
 func GetPgFetcher(dbDataDirectory, fileMask, restoreSpecPath string) func(folder storage.Folder, backup Backup) {
 	return func(folder storage.Folder, backup Backup) {
 		filesToUnwrap, err := backup.GetFilesToUnwrap(fileMask)
-		tracelog.ErrorLogger.FatalfOnError("Failed to fetch backup: %v\n", err)
+		tracelog.ErrorLogger.FatalfOnError("Failed to fetch backup: %v", err)
 
 		var spec *TablespaceSpec
 		if restoreSpecPath != "" {
 			spec = &TablespaceSpec{}
 			err := readRestoreSpec(restoreSpecPath, spec)
-			errMessege := fmt.Sprintf("Invalid restore specification path %s\n", restoreSpecPath)
+			errMessege := fmt.Sprintf("Invalid restore specification path %s: %%v", restoreSpecPath)
```

Comment:
There is only 1 argument.

Reply:
yes, but `%%v` will be `%v` after `Sprintf`, and later it  is used in `FalalfOnError`.